cypress/integration/tests/controls/stakeholders/crud.test.ts

```diff
@@ -140,16 +148,16 @@ describe("Stakeholder CRUD operations", () => {
 
         // Delete stakeholder
         stakeholder.delete();
+        cy.wait("@deleteStakeholder");
         cy.wait("@getStakeholders");
 
         // Assert that newly created stakeholder is deleted
-        stakeholder.notExists();
+        notExists(stakeholder.email);
 
-        // Delete jobfunctions and groups created at the start of test
+        // Perform clean up by deleting jobfunctions and groups created at the start of test
         jobfunctions.forEach(function (jobfunction) {
```

Comment:
As we are working more towards making structure more robust. 
can we think about `after` block for cleanup of this test case?

Reply:
Yes, need to check access of variables in after block.